### DISCUSSION TITLE
I have some difficulty in understanding the document, Is there a problem with my reading style?

### DISCUSSION THREAD

**Discussion by Idletalk** [fabricjs/fabric.js#9155] [Q&A] [answered]
Take this case for example, I want to create a Line, however, there are only two for the parameters "points" and "options", but "options" is not clear what the options include?
![image](https://github.com/fabricjs/fabric.js/assets/38195804/294a5ded-843e-45e8-9427-35655d9f43ce)

**Comment by ShaMan123** [comment, marked as answer]
The docs are outdated unfortunately
The new website in in development
I suggest for now using v6 and autocomplete by vscode

**Reply by Idletalk** [reply]
Thank you for your reply, looking forward to the new website, is there any way to get the progress of the new website, or is there an approximate time

**Reply by ShaMan123** [reply]
If you want to help out ping @asturur
